docs/internal-port.md

```diff
@@ -35,6 +35,5 @@ metadata:
 ```
 
 ## Some limitation
-The internal port name should be unique on a host and kubelet always check the `eth0` interface in the Pod.
-
-To bypass this issue, Kube-OVN creates a dummy type device in Pod netns with the same ip address of internal port and set the eth0 down. It works well for most scenarios, however if applications rely on network interface name, it will bring confusions.
+- The internal port name should be unique on a host and kubelet always check the `eth0` interface in the Pod. To bypass this issue, Kube-OVN creates a dummy type device in Pod netns with the same ip address of internal port and set the eth0 down. It works well for most scenarios, however if applications rely on network interface name, it will bring confusions.
+- When OVS restart, the internal port will be removed from the pod. Pods on that node with internal-port type nic should be restarted manually.
```

Comment:
After OVS restarts, internal ports will be deattach from the pod. Pods on the same node with internal-port interfaces should be recreated manually.